nifi-assembly/LICENSE

```diff
@@ -208,6 +208,77 @@ notices and license terms. Your use of the source code for the these
 subcomponents is subject to the terms and conditions of the following
 licenses. 
 
+This product bundles 'Apache Ant 1.9.13' which is available under an ALSv2 license and includes 'W3C'.
+
+    W3C� SOFTWARE NOTICE AND LICENSE
```

Comment:
That character is in the LICENSE that I pulled in from groovy-ant-2.5.4.jar.  This commit is also from [PR 3547](https://github.com/apache/nifi/pull/3547), which has since been merged to master.  The `®` symbol is what that's supposed to be, and it is part of the actual license.  That could be replaced with the ASCII representation of that symbol.